live-examples/js-examples/globalprops-decodeuri.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">var uri = "https://mozilla.org/?x=шеллы";
+var encoded = encodeURI(uri);
+console.log(encoded);
+// expected output: https://mozilla.org/?x=%D1%88%D0%B5%D0%BB%D0%BB%D1%8B
+
+try {
+  console.log(decodeURI(encoded));
+} catch(e) { // catches a malformed URI
+  console.error(e);
```

Comment:
Just a note here. We are not capturing `console.error` so, if someone plays around with this to test the `try/catch` functionality, they will not see the `console.error` unless they have their dev tools open :-/

Perhaps this is something we need to also address? @wbamberg

Reply:
Good to know. Happy to adjust if needed.

Reply:
I think we can omit support for most of the [console API](https://developer.mozilla.org/en-US/docs/Web/API/console) (and some would be difficult).

But it would be nice to have `console.error()`, and it doesn't seem like it would be too difficult.

Reply:
`console.error` support has landed on master ~ https://github.com/mdn/interactive-examples/pull/279

Reply:
woohoo!